pkg/apis/nodeup/bootstrap.go

```diff
@@ -35,7 +35,7 @@ type BootstrapRequest struct {
 // BootstrapResponse is a response to a BootstrapRequest.
 type BootstrapResponse struct {
 	// Certs are the issued certificates.
-	Certs map[string]string
+	Certs map[string]string `json:"certs,omitempty"`
```

Comment:
In general, nodes can only bootstrap off of kops-controllers of the same version because otherwise the hashes won't match. So there's no real problem.

This change isn't really necessary. The protocol is internal; it doesn't have to look pretty.

Reply:
`json:"Certs,omitempty"` seems reasonable for now. We can have a cleaner version later.

Reply:
The `omitempty` seems superfluous, but whatever.